examples/cpp/BasicOperations/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
BasicOperations name stinks of the old quick starts. Avoid camel case. Come up with a better name for this example.